apps/src/componentLibrary/radioButton/radioButton.module.scss

```diff
@@ -0,0 +1,164 @@
+@import "color";
+
+.radioButton {
+  display: flex;
+  gap: 10px;
+  align-items: center;
+
+  input[type="radio"] {
+    position: absolute;
+    opacity: 0;
+  }
+
+  i {
```

Comment:
Can these rules be applied via classnames?

Reply:
Agree, it might improve readability of styles. Updated